cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -91,7 +91,7 @@ module Cardano.Api.Typed (
     makeByronTransaction,
     makeShelleyTransaction,
     SlotNo,
-    TxOptional(..),
+    TxExtraContent(..),
```

Comment:
Do you like this name? I'm unsure.

I know I was the one who changed it :-)

Reply:
The only other name I can think of is `TxAdditional`. Both work for me.

Reply:
It's not yet final, it's fine. We can still decide what colour we like best.